mjolnir/input/read_local_potential.hpp

```diff
@@ -362,6 +385,48 @@ read_local_potentials(const toml::value& local)
     return retval;
 }
 
+template<std::size_t N, typename realT, typename angle1_potentialT,
+         typename angle2_potentialT, typename contact_potentialT>
+std::vector<std::tuple<std::array<std::size_t, N>, angle1_potentialT,
+                       angle2_potentialT, contact_potentialT>>
+read_directional_contact_potentials(const toml::value& local)
```

Comment:
This function is too specific to call from other interactions. Is there a reason to split it from `read_directional_contact_interaction` stuff?

Reply:
The process corresponds to this function is implemented in read_directional_contact_iteraction more than once. Considering the amount of code for this process, first of all, I thought splitting code is better for readability.

Reply:
Okay, but still, I feel this function is too specific. Isn't it good to move it to `read_interaction.hpp` because this function does not make sense without knowledge of the interaction.